### ISSUE TITLE
Support vty-6 and brick-2.

### ISSUE THREAD

**Issue by amano-kenji** [codedownio/sandwich#82]
brick-2 depends on vty-6 which introduces breaking changes.

From vty-6, you have to depend on `vty-crossplatform` or `vty-unix` or `vty-windows` for `mkVty` function.

**Comment by thomasjm** [comment]
Already done in https://github.com/codedownio/sandwich/tree/vty-6 branch.

New release coming soon.

**Comment by amano-kenji** [comment]
Okay.

**Comment by thomasjm** [comment]
Just released in [0.2.0.0](https://hackage.haskell.org/package/sandwich-0.2.0.0)
